package.json

```diff
@@ -43,6 +43,7 @@
         "@testing-library/dom": "^6.0.0",
         "@testing-library/jest-dom": "^5.1.1",
         "@testing-library/react": "^9.1.1",
+        "@testing-library/react-hooks": "^3.2.1",
```

Comment:
@piofinn la til denne og react-test-renderer for å kunne teste hooks direkte.

Reply:
Smud!